### PR TITLE
Towards #3041 - Use new Task for counts and some other things

### PR DESCRIPTION
**Pull request by kolloch**

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/core/appinfo/TaskLifeTime.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@ case class TaskLifeTime(
   medianSeconds: Double)
 
 object TaskLifeTime {
-  def forSomeTasks(now: Timestamp, tasks: Iterable[MarathonTask]): Option[TaskLifeTime] = {
+  def forSomeTasks(now: Timestamp, tasks: Iterable[Task]): Option[TaskLifeTime] = {
```
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/core/appinfo/TaskLifeTime.scala]: if there is a `forSomeTasks`, will there be a `forALotOfTasks`?

Comment (kolloch) [inline src/main/scala/mesosphere/marathon/core/appinfo/TaskLifeTime.scala]: Not part of this PR but I admit your comment is funny ;)

Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/core/appinfo/TaskLifeTime.scala]: My suggestion would be to rename it: `forTasks`

**Comment by aquamatthias** [comment]
LGTM
